deploy/charts/trust-manager/values.yaml

```
@@ -292,7 +292,7 @@ app:
           #     my-secret-annotation-1: "foo"
           #     my-secret-annotation-2: "bar"
           #   labels:
-          #     my-secret-label: foo
+        #     my-secret-label: foo
```

```suggestion
          #     my-secret-label: foo
```

No need to change this.